docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -39,32 +38,23 @@ bigchaindb -y configure rethinkdb
 Show the values of the [BigchainDB node configuration settings](configuration.html).
 
 
-## bigchaindb export-my-pubkey
-
-Write the node's public key (i.e. one of its configuration values) to standard output (stdout).
-
-
 ## bigchaindb init
 
-Create a backend database (RethinkDB or MongoDB),
-all database tables/collections,
-various backend database indexes,
-and the genesis block.
+Create a backend database (Local MongoDB), all database tables/collections,
+various backend database indexes, and the genesis block.
```

Comment:
I'm not sure if Tendermint wants us to create a Genesis block. We can leave this text alone for now, but I created a new issue #2094 to figure out what to do with our code to create a Genesis block.